# --- T2-COPYRIGHT-NOTE-BEGIN ---
# This copyright note is auto-generated by ./scripts/Create-CopyPatch.
# 
# T2 SDE: package/.../sun-jdk-142/extract-ia64.diff
# Copyright (C) 2004 - 2005 The T2 SDE Project
# 
# More information can be found in the files COPYING and README.
# 
# This patch file is dual-licensed. It is available under the license the
# patched project is licensed under, as long as it is an OpenSource license
# as defined at http://www.opensource.org/ (e.g. BSD, X11) or under the terms
# of the GNU General Public License as published by the Free Software
# Foundation; either version 2 of the License, or (at your option) any later
# version.
# --- T2-COPYRIGHT-NOTE-END ---
--- ../j2sdk-1_4_2_08-linux-ia64.bin	2005-04-14 16:56:07.000000000 +0200
+++ j2sdk-1_4_2_08-linux-ia64.bin	2005-04-14 18:46:00.000000000 +0200
@@ -11,7 +11,7 @@
 localinstall=$1
 diskSpaceRequired=124672
 MIME_TYPE=application/x-java-jnlp-file
-more <<"EOF"
+cat <<"EOF"
                 Sun Microsystems, Inc.
              Binary Code License Agreement
                      for the
@@ -341,7 +341,7 @@
 Network Circle, Santa Clara, California 95054, U.S.A.
 (LFI#129530/Form ID#011801)
 EOF
-agreed=
+agreed=1
 while [ x$agreed = x ]; do
     echo
     echo "Do you agree to the above license terms? [yes or no] "
@@ -363,7 +363,7 @@
 fi
 trap 'rm -f $outname; exit 1' HUP INT QUIT TERM
 echo "Unpacking..."
-tail +511 $0 > $outname
+tail -n +511 $0 > $outname
 if [ -x /usr/bin/sum ] ; then
     echo "Checksumming..."
 
